products/customer_analytics/frontend/components/ZendeskTicketsFilters/OrderBy.tsx

```diff
@@ -0,0 +1,48 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonSelect } from '@posthog/lemon-ui'
+
+import { capitalizeFirstLetter } from 'lib/utils'
+
+import { zendeskTicketsFiltersLogic } from './zendeskTicketsFiltersLogic'
+
+export const OrderBy = (): JSX.Element => {
+    const { orderBy, orderDirection } = useValues(zendeskTicketsFiltersLogic)
+    const { setOrderBy, setOrderDirection } = useActions(zendeskTicketsFiltersLogic)
+
+    const options = ['updated_at', 'created_at']
+
+    const label = (key: string) => {
+        switch (key) {
+            case 'updated_at':
+                return 'Updated'
+            case 'created_at':
+                return 'Created'
+            default:
+                return capitalizeFirstLetter(key)
+        }
+    }
```

Comment:
these could be outside the component to avoid re-creating the function every render